packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -42,4 +73,105 @@ export class CartCheckoutPage {
 		await this.page.click( selectors.removeCartItemButton( cartItemName ) );
 		await this.page.click( selectors.modalContinueButton );
 	}
+
+	/**
+	 * Obtains the text content of the payment button.
+	 *
+	 * @returns {string} Content of the payment button.
+	 */
+	async getPaymentButtonText(): Promise< string > {
+		const elementHandle = await this.page.waitForSelector( selectors.paymentButton );
+		return await elementHandle.innerText();
+	}
+
+	/**
+	 * Apply a coupon.
+	 *
+	 * If a coupon code is supplied as parameter, this method will apply the supplied code.
+	 * Otherwise, this method will apply the default coupon code found in the configuration.
+	 *
+	 * @param {string} coupon Custom coupon code.
+	 */
+	async enterCouponCode( coupon?: string ): Promise< void > {
+		await this.page.click( selectors.couponCodeInputButton );
+
+		const couponCode =
+			coupon !== undefined ? coupon : ( DataHelper.config.get( 'testCouponCode' ) as string );
+
+		await this.page.fill( selectors.couponCodeInput, couponCode );
+		await this.page.click( selectors.couponCodeApplyButton );
+		await this.page.click( selectors.dismissBanner );
+	}
+
+	/**
+	 * Returns the total amount of items in cart.
+	 *
+	 * @returns {string} Total value of items in cart.
```

Comment:
Do we need to make this i18n safe? In the cookies function, you allow for other currencies, but this current function wouldn't be able to some i18n currency options (mostly due to separator differences).

I think that's fine if we don't want to for now, but we should make that very clear in the docs or function name that it may not work for all currency separators!

Reply:
My guess is you're talking about continental Europe where they use a full stop for separating thousands and a comma for the decimal place?

You are correct, that would mess this up. I don't know of a good way to handle that - honestly it's a mess (digit grouping) and I wish we just didn't do it.

Reply:
My proposal is to leave this to a future time when that becomes necessary.

In our testing so far, we haven't used anything other than GBP and AUD. There is a point to be made about testing other currencies, but let's cross that bridge when we need to.